### DISCUSSION TITLE
photo Interrupter Sensor

### DISCUSSION THREAD

**Discussion by saftione** [tttapa/Control-Surface#499] [Q&A]
Hey, 
is it possible to use an photo Interrupter Sensor like LM393 Speed sensor ?

**Comment by tttapa** [comment]
What do you mean by "use"? You can connect it to an Arduino and read the pulses, which you could use in your code to send MIDI messages, but this is not something that Control Surface has ready-made examples for.

**Reply by saftione** [reply]
yes, thx for your feedback. 
I wanted to know if its somehow included already. 
I have an old dj controller and they have no encoder but a photo interrupter in use
it is still a bit too tricky for me to code this myself
